content/soarian/dstu2/soarian_authorization.md

```diff
@@ -0,0 +1,119 @@
+  
+For details on above steps refer to [Authorization] [authorization-behalf-user]
+Soarian supports only SMART Application Launch flow as listed in the [technical specification] [smart-application]
+Soarian supports only "patient" scope for the resources specified in the Conformance statement. (For example: patient/Condition.read, patient/MedicationStatement.read etc.)
+
+## ActivateLinkToken API to link User Identity to Patient
+
+### Overview
+
+Soarian Clinicals generates and provides Activation tokens for each of the patients and/or patient's proxies who wish to request patient's data and the Application needs to provide a means to enter Activation tokens and call ActivateLinkToken API to link the user to the patient record (user is linked as a patient or a defined patient proxy).  
+
+Activation tokens can be provided to patients and their proxies. Soarian Clinicals allows hospital staff to add patient proxies and manage their tokens. This Activation token is provided to patient and/or their proxies by hospital staff either by printed report or email, based on hospital's configuration and policy.  Activation tokens are temporary and expire if not utilized; the API will error if a token is invalid, revoked or expired.  Patients can request hospital staff to generate another Activation token, if needed.  
+
+API-consuming Application(s) need to invoke ActivateLinkToken API to establish a link between the established user identity and a patient record in Soarian Clinicals. This service when called, the Application must provide Authorization token (access token received from Cerner OAuth server), valid Activation token (provided by Hospital to Patient or Proxy) and Patient's Date of Birth. This ActivateLinkToken API to link the user with a patient record is a one-time action and must be called prior to making any API calls for other FHIR resources supported by Cerner Soarian Clinicals.  The user to patient link established remains valid until the time it is manually revoked by hospital staff on patient request. 
+
+### Details
+The following provides information about the ActivateLinkToken API.
+
+### API Root Url
+
+The root URL for this API would be as follows:   
+Sandbox: https://api-myrecord.sandboxcerner.com/soarian/\<tenantID>
+
+Production: https://api-myrecord.cerner.com/soarian/\<tenantID>
+
+###	Requesting OAuth Access Token
+
+ActivateLinkToken API requires OAuth access token from Cerner OAuth authorization server. Cerner has designed a special profile to access this API.  Each Soarian Clinicals instance advertises the URL of its respective authorization server within its FHIR<sup>&reg;</sup> [Conformance] [conformance] document for the ActivateLinkToken API under 
+ 
+	"rest": [
+    {
+      "mode": "server",
+      "security": {
+        "extension": [ ...
+            ]
+          },
+          {
+            "url": "http://fhir-registry.smarthealthit.org/StructureDefinition/SoarianIdLinking-uris",
```

Comment:
This JSON also isn't currently valid. I would guess it should be updated to something like:

```
    "rest": [
      {
        "mode": "server",
        "security": {
          "extension": [
            ...
          ]
        },
        {
          "url": "http://fhir-registry.smarthealthit.org/StructureDefinition/SoarianIdLinking-uris",
          "extension": [
            {
              "url": "token",
              "valueUri": "https://authorization.sandboxcerner.com/tenants/45343c0c-3f93-4721-874a-  258fc4bae3a4/protocols/oauth2/profiles/soarian-identity-linking-v1/token"
            },
            {
              "url": "authorize",
              "valueUri": "https://authorization.sandboxcerner.com/tenants/45343c0c-3f93-4721-874a-  258fc4bae3a4/protocols/oauth2/profiles/soarian-identity-linking-v1/personas/patient/authorize"
            }
          ]
        },
        "description": "OAuth2 with SMART extensions"
      },
      ...
    ]
```

(the closing square bracket on line 81 doesn't align with an opening square bracket earlier)